src/constants/api-keys.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  androidSafetyNetApiKey:
+    process.env['ANDROID_SAFETY_NET_API_KEY'] ?? 'testEnvFallback'
```

Comment:
Did you test this on runtime?

When reading the CI files, it seems like you are defining `ANDROID_SAFETY_NET_API_KEY` env variable for the build process (`Build Android App Bundle` task), but are you sure this will be set also for the runtime process?

Reply:
I figured we only needed it at build time, I shall check if the variable is injected in runtime

Reply:
`ANDROID_SAFETY_NET_API_KEY` is used on runtime during the login process. The mobile should do the flagship certification and call Google APIs.

Reply:
using react-native-config to inject the secret into runtime as well as using it directly during build time